lib/logthrdestdrv.c

```diff
@@ -220,9 +220,6 @@ _process_result(LogThreadedDestDriver *self, gint result, LogMessage *triggering
 
       if (self->retries.counter >= self->retries.max)
         {
-          if (self->messages.retry_over && triggering_message)
-            self->messages.retry_over(self, triggering_message);
```

Comment:
Is there any other way to know which message is dropped?

Reply:
I would prefer to add this mechanism when the need arises. I am not sure this interface is that good at solving this issue. I mean the insert() function already knows that  a message would be dropped, because it is the one that decides that there was an error. The retry mechanism would probably not be needed if someone has an alternative location.

I was actually looking at what used this interface and there were two locations:
 1) the python binding: forwarded to the python code
 2) mongodb: to log the details of the message.

If we wanted to retain this function, then probably this would have to become batch based, as we are retrying batches and not just messages. 

Anyway, back to my initial point: I would wait for an actual use-case to surface.

Is this ok?